lib/components/ViewWalletButton/index.tsx

```diff
@@ -0,0 +1,32 @@
+import { cx } from "@emotion/css";
+import { Launch } from "@mui/icons-material";
+import React, { FC } from "react";
+
+import { ButtonPrimary } from "../";
+import { concatAddress } from "../../utils";
+import { ButtonBaseProps } from "../Buttons/ButtonBase";
+import * as styles from "./styles";
+
+type Props = {
+  address: string;
+} & ButtonBaseProps;
+
+export const ViewWalletButton: FC<Props> = (props) => {
+  const className = cx(styles.viewWalletButton, props.className);
+
+  return (
+    <div className={className}>
+      <span>{concatAddress(props.address)}</span>
+      <ButtonPrimary
+        href="https://polygon.tor.us/"
```

Comment:
I would pass this url as a prop to make this "ViewWalletButton" compatible with future wallets too

Reply:
good point, done!